simplefs/simplefs.go

```diff
@@ -164,6 +165,24 @@ func rawPathFromKbfsPath(path keybase1.Path) (string, error) {
 	}
 }
 
+func rawPathFromPath(path keybase1.Path) (string, error) {
```

Comment:
If you take my advice on the other PR, you won't need this function, since the target will just be a string.